pkg/storage/filesystem/btrfs.go

```diff
@@ -516,10 +516,19 @@ func (v *btrfsVolume) Usage() (usage Usage, err error) {
 		return
 	}
 
+	size := group.MaxRfer
+	if size == 0 {
+		// in case no limit is set on the subvolume, we assume
+		// it's size is the size of the files on that volumes
+		size, err = FilesUsage(v.Path())
```

Comment:
Is this a viable solution for a multi TB disk ? I'm afraid this will be quite slow ?
@delandtj ?

Reply:
Unfortunately this is the only accurate way.